deps/build.jl

```diff
@@ -40,6 +40,20 @@ end
 info("Found Jupyter version $jupyter_vers: $jupyter")
 
 #######################################################################
+# Get the latest syntax highlighter file.
+if isconda
+    highlighter = joinpath(Conda.LIBDIR, "python2.7", "site-packages", "notebook", "static",
+                           "components", "codemirror", "mode", "julia", "julia.js")
+    # CodeMirror commit from which we get the syntax highlighter
+    cm_commit = "d444cd1007bd32e418ace8ffed1cb0e692ee147d"
+    highlighter_url = "https://raw.githubusercontent.com/codemirror/CodeMirror/" *
+                      cm_commit * "/mode/julia/julia.js"
+    try
+        isfile(highlighter) && download(highlighter_url, highlighter)
```

Comment:
Not really necessary, but if you wanted to get really fancy, you could do something like
```julia
if isfile(highlighter)
    try
        download(highlighter_url, highlighter)
    catch
        info("Unable to download latest syntax highlighting definitions. ",
             "Your definitions may be out of date.")
    end
end
```